src/changeset.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import unicode_literals
+
+import itertools
+
+from models import (
+    Relation,
+    UnitPlacement,
+)
+
+
+def _parse_v3_unit_placement(placement):
+    """Return a UnitPlacement for bundles version 3, given a placement string.
+    """
```

Comment:
Could you include a reference or definition of placement string here or perhaps in the module level docstring for the uninitiated?